inc/woocommerce.php

```diff
@@ -59,3 +59,15 @@ function mollieWooCommerceOrderKey(WC_Order $order)
         : $order->get_order_key();
 }
 
+/**
+ * Mimics wc_string_to_bool
+ * @param $string
+ *
+ * @return bool
+ */
+function mollieWooCommerceStringToBoolOption($string)
```

Comment:
Would be better to use the real function `wc_string_to_bool` name and body, wrap it in a conditional. 
That would save you in future when the Wc supported version will rise and you have to keep this function instead of the real one.